src/firefly/java/edu/caltech/ipac/firefly/server/WebDAVWorkspaceManager.java

```diff
@@ -427,34 +446,6 @@ public WspaceMeta getMeta(String relPath) {
         return getMeta(relPath, WspaceMeta.Includes.ALL);
     }
 
-    public WsResponse search(String relPath) throws IOException {
-        String query = "" +
-                "  " +
-                " SELECT *" +
-                "  " +
-                "";
-        query = "//element(*,rep:root)";
-
-        OptionsMethod options = new OptionsMethod(getResourceUrl(relPath));
-        executeMethod(options);
-        String okMethod = "SEARCH";
-        if (!options.isAllowed(okMethod)) {
-            String s = "";
-            String[] allowedMethods = options.getAllowedMethods();
-            for (String method : allowedMethods) {
-                s += method + "\n";
-            }
-            return WsUtil.error(options.getStatusCode(), options.getStatusText(), okMethod + " is not allowed - only:\n" + s);
-        }
-        SearchMethod m = new SearchMethod(getResourceUrl(relPath), query, "xpath");
-        if (!executeMethod(m)) {
-            // handle error
-            LOG.error("Unable to move:" + relPath + " -- " + m.getStatusText());
-            return WsUtil.error(m.getStatusCode(), m.getStatusLine().getReasonPhrase());
-        }
-        return WsUtil.success(m.getStatusCode(), m.getStatusText(), relPath);
-    }
-
     public WspaceMeta getMeta(String relPath, WspaceMeta.Includes includes) {
```

Comment:
Is there a way to get a set of props for all the files. i.e. owner?

Reply:
yes, getMeta("/", ALL). See unit test or I can explain later when we meet again.